tests/mtgjson5/providers/test_enrichment_provider.py

```diff
@@ -0,0 +1,259 @@
+"""Test the EnrichmentProvider."""
```

Comment:
I greatly appreciate you included tests! Hopefully these help with any revisions we might come to an agreement on